var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -866,7 +879,10 @@ def cmake_args(self):
             else:
                 gpu_ver = gpu_map[spec.variants["cuda_arch"].value[0]]
                 args += ["-DCP2K_USE_ACCEL=CUDA"]
-                args += [self.define("CP2K_WITH_GPU", gpu_ver)]
+                args += [
+                    self.define("CP2K_WITH_GPU", gpu_ver),
+                    self.define_from_variant("CP2K_USE_CUSOLVERMP", "cusolvermp"),
+                ]
```

Comment:
Worth adding the `-DCP2K_USE_ACCEL=CUDA` option above to the same list for consistency?